providers/default.rb

```diff
@@ -201,26 +201,42 @@ def windows?
 # gem files.
 #
 def move_opt_chef(src, dest)
-  converge_by("move all files under #{src} to #{dest}") do
-    FileUtils.rm_rf dest
-    raise "rm_rf of #{dest} failed" if ::File.exist?(dest) # detect mountpoints that were not deleted
-    FileUtils.mv src, dest
-  end
-rescue => e
-  # this handles mountpoints
-  converge_by("caught #{e}, falling back to copying and removing from #{src} to #{dest}") do
+  if windows?
+    powershell_script 'move src -> using powershell' do
```

Comment:
so this isn't going to work on chef-11, which may kill this idea.  perhaps the windows cookbook could be used, but i really fear opening up dependencies on other cookbooks, possibly old versions of cookbooks, and old bugs in old versions of cookbooks, and the windows cookbook in particular...

Reply:
This is not true. The resource powershell_script was already supported in 11.6 according to this doc:
https://docs-archive.chef.io/release/11-6/resource_powershell_script.html